patches/tModLoader/Terraria/ModLoader/ModContent.cs

```diff
@@ -462,6 +467,7 @@ private static void SetupRecipes(CancellationToken token)
 	internal static void UnloadModContent()
 	{
 		MenuLoader.Unload(); //do this early, so modded menus won't be active when unloaded
+		CloudLoader.Unload();
```

Comment:
These usually go in ModContent.Unload

Reply:
Clouds are present on the menu, if you don't unload early it crashes.